src/cache/raw.rs

```diff
@@ -118,22 +120,43 @@ pub(crate) struct Member {
     ///
     /// `0` means `false`, all other values mean `true`.
     pub(crate) is_synthesized: u8,
-
+    /// Whether this member refers to an outline method.
+    ///
+    /// `0` means `false`, all other values mean `true`.
+    pub(crate) is_outline: u8,
+    /// Offset into the outline pairs section for this member's outline callsite mapping.
+    pub(crate) outline_pairs_offset: u32,
+    /// Number of outline pairs for this member.
+    pub(crate) outline_pairs_len: u32,
```

Comment:
For struct alignment/size reasons, the `u32` members should come before the `u8` members, i.e. `outline_pairs_offset` and `outline_pairs_len` should be moved before `is_synthesized`.